catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -94,50 +79,61 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
     [images]
   );
 
-  const imageRenderer = useCallback(galleryImage => {
-    const photo: GalleryImageProps = galleryImage.photo;
+  const imageRenderer: React.FC<RenderPhotoProps<GalleryImageProps>> = ({
+    photo,
+    layout,
+    wrapperStyle,
+  }) => {
     const rgbColor = hexToRgb(photo.averageColor || '');
-
     return (
-      <ImageContainer key={galleryImage.key} role="listitem">
-        <ImageCard
-          id={photo.id}
-          workId={photo.source.id}
-          image={{
-            contentUrl: photo.src,
-            width: photo.width - imageMargin * 2,
-            height: photo.height,
-            alt: photo.source.title,
+      <div style={wrapperStyle}>
+        <div
+          style={{
+            display: 'block',
+            position: 'relative',
+            width: '100%',
+            height: '100%',
           }}
-          onClick={event => {
-            event.preventDefault();
-            setExpandedImage(photo);
-            setIsActive(true);
-          }}
-          layout="fixed"
-          background={
-            background ||
-            (rgbColor &&
-              `rgba(${rgbColor.r}, ${rgbColor.g}, ${rgbColor.b}, 0.5)`)
-          }
-        />
-      </ImageContainer>
+        >
+          <ImageCard
+            id={photo.id}
+            workId={photo.source.id}
+            image={{
+              contentUrl: photo.src,
+              width: layout.width,
+              height: layout.height,
+              alt: photo.source.title,
+            }}
+            onClick={event => {
+              event.preventDefault();
+              setExpandedImage(photo);
+              setIsActive(true);
+            }}
+            layout="fixed"
+            background={
+              background ||
+              (rgbColor &&
+                `rgba(${rgbColor.r}, ${rgbColor.g}, ${rgbColor.b}, 0.5)`)
+            }
+          />
+        </div>
+      </div>
     );
-  }, []);
+  };
 
   return (
     <>
       {isFullSupportBrowser && !isSmallGallery && (
-        <PlainList role="list">
```

Comment:
Why remove the list/should search results be in a list? (either way we go with, we could look at closing or doing this ticket https://github.com/wellcomecollection/wellcomecollection.org/issues/8919)